release notes/v0.57.0.md

```diff
@@ -0,0 +1,180 @@
+k6 `v0.57.0` is here 🎉! This release includes:
+
+- Introducing helpers for functional testing.
+- The `k6/experimental/csv` module gets a new `asObjects` option.
+- We no longer support the `k6/experimental/browser` module.
+- Moving all non-public APIs to the `internal` package.
+- Many other improvements, bug fixes, and internal refactors.
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+
+## Breaking changes
+
+- [#4161](https://github.com/grafana/k6/pull/4161) Drops `k6/experimental/browser`. If you are still using it, please follow the [instructions](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) to move to the not experimental module.
+- [#4133](https://github.com/grafana/k6/pull/4133) Moves all not publicly used API in `internal` package. This was based on the publicly available extensions for k6. And may break private ones. There is a good chance more APIs will be removed or changed in follow-up releases after this more mechanical change.
+- [#4292](https://github.com/grafana/k6/pull/4292) Makes type stripping on by default and deprecates `experimental_enhanced` compatiblity-mode as it is no longer necessary. This means that now files with `.ts` extension will automatically work with k6.
+
+### (_optional h3_) `<big_breaking_change>` `#pr`
+
+## New features
+
+_optional intro here_
+
+### `<big_feature_1>` `#pr`
+
+_what, why, and what this means for the user_
+
+### New functional testing focused official jslib [k6-testing](https://github.com/grafana/k6-jslib-testing)
+
+The k6 team has been developing a new official jslib dedicated to functional testing. While it is still under active development and will potentially see breaking changes, the set of APIs and behaviors it offers are meant to make their way into k6 eventually, and it is now available for early feedback.
+
+[k6-testing](https://github.com/grafana/k6-jslib-testing) is a k6 javascript library that offers a seamless way to write functional tests in k6, using a Playwright-compatible assertions API. It exposes an `expect` function, with which assertions can be performed using specific matchers that reflect the expectation.
```

Comment:
```suggestion
[k6-testing](https://github.com/grafana/k6-jslib-testing) is a k6 JavaScript library that offers a seamless way to write functional tests in k6, using a Playwright-compatible assertions API. It exposes an `expect` function, with which assertions can be performed using specific matchers that reflect the expectation.
```

Reply:
"expectation" at the end seems a little bit odd to me. What if we replaced it with "expected results"?

Reply:
```suggestion
[k6-testing](https://github.com/grafana/k6-jslib-testing) is a k6 JavaScript library that offers a seamless way to write functional tests in k6, using a Playwright-compatible assertions API. It exposes an `expect` function, with which assertions can be performed using specific matchers that reflect the expected results.
```